apparmor.d/groups/apps/discord

```diff
@@ -40,5 +48,12 @@ profile discord @{exec_path} {
 
   owner @{run}/user/@{uid}/discord-ipc-@{int} rw,
 
+  owner @{user_config_dirs}/discord/** rwm,
+
+  @{PROC}/@{pid}/task/@{tid}/status r,
+
+  # not actually required
+  # owner /dev/pts/@{int} rw,
```

Comment:
If it is not needed, do not put it here. Otherwise that would be added using the `consoles` abstraction.

Reply:
That's why I left it. The logs show the permission to be used but without it the app runs just fine.
I will remove it entirely to make it clean